packages/blade/src/components/ChatBubble/SelfMessageBubble.web.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import BaseBox from '~components/Box/BaseBox';
+import { AlertCircleIcon } from '~components/Icons';
+import { Text } from '~components/Typography';
+
+const SelfMessageBubble = ({
+  children,
+  isError,
+  errorText = 'Message not sent. Tap to retry.',
+  onClick,
+  messageType,
+}: {
+  children: React.ReactNode | string;
+  isError?: boolean;
+  errorText?: string;
+  onClick?: () => void;
+  messageType: 'last' | 'default';
+}): React.ReactElement => {
```

Comment:
type SelfMessageBubbleProps = {} 

And also keep internal component's types similar to the main component such that we can just do `SelfMessageBubbleProps = Pick<CommonProps, 'children' | 'validationState' | 'errorText'....>`